test/test_helper.rb

```diff
@@ -38,5 +38,5 @@ def sign_in(role = :editor)
 
 def fill_in_form(article = :ror)
   fill_in('Title', with: articles(article).title)
-    fill_in('Body', with: articles(article).body)
+  fill_in('Body', with: articles(article).body)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.